helm-charts/policy/templates/policy_framework_cluster_role.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.org }}:{{ template "policy.fullname" . }}-framework
+  labels:
+    app: {{ template "policy.name" . }}-framework
+    chart: {{ template "policy.chart" . }}
+    component: "policy"
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  - namespaces
+  verbs:
+  - create
+  - delete
```

Comment:
Could you please verify if deleting a namespace is required? I'd feel more comfortable if that wasn't part of the controller's ACLs.

Also being able to delete events raises a red flag.

Reply:
Yes, we do need to delete the cluster namespace as this namespace is created and only used by policy framework. Maybe we should limit to just allow deleting the namespace with cluster name.